core/templates/dev/head/components/summary-tile/topic-summary-tile.directive.html

```diff
@@ -6,7 +6,8 @@
     <div class="oppia-topic-name">
       <[::$ctrl.getTopicSummary().getName()]>
     </div>
-    <[::$ctrl.getTopicSummary().getCanonicalStoryCount()]> lessons
+    <[::$ctrl.getTopicSummary().getCanonicalStoryCount()]>
+    lesson<span ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() !== 1">s</span>
```

Comment:
The word "lesson"/"lessons" needs to be internationalized, but you can do it in a separate PR (please file an issue).

Reply:
Created #7553